src/BridgeIHMCORS.h

```diff
@@ -20,10 +20,20 @@
 #include <robotDesired.h>
 #include <robotFeedback.h>
 
+// socket and port communication
+#include <iostream>
+#include <asio.hpp>
+
+// fast_cdr includes for serialization
+#include <fastcdr/FastBuffer.h>
+#include <fastcdr/Cdr.h>
+
+using namespace eprosima::fastcdr;
+using asio::ip::udp;
```

Comment:
Not a big problem as this header will only be used in the BridgeIHMCORS.cpp compilation unit, but in general it is better to avoid using the `using` directives in headers (https://stackoverflow.com/questions/5849457/using-namespace-in-c-headers) .